### ISSUE TITLE
Example 2 Step 5 Offset Issue

### ISSUE THREAD

**Issue by ctoestreich** [confluentinc/qcon-ai-workshop#2]
Due to the fact that all data was loaded in the previous example and was not removed, the `get with avro schema offset` is wrong and will error on bad data.

The offset here is likely wrong given you came right from Exercise 1.

```
curl -H "Accept: application/vnd.kafka.avro.v1+json" http://localhost:8082/topics/movies-raw/partitions/0/messages?offset=0&count=10
```

Prior to Exercise 2 it might be warranted to drop and recreate the topic or instructions to start clean or check offset and augment rest call with offset count.  Also could create and use new movies-raw-avro topic...

**Comment by gwenshap** [comment]
You are right :) And also, after exercise 2...
We are fixing this right now.

**Comment by gwenshap** [comment]
Resolved in commit: a04df36
